packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -712,13 +712,12 @@ function createInstances(cluster: DatabaseClusterNew, props: DatabaseClusterBase
       deleteAutomatedBackups: props.instanceProps.deleteAutomatedBackups,
     });
 
-    // If removalPolicy isn't explicitly set,
-    // it's Snapshot for Cluster.
-    // Because of that, in this case,
-    // we can safely use the CFN default of Delete for DbInstances with dbClusterIdentifier set.
-    if (props.removalPolicy) {
-      applyRemovalPolicy(instance, props.removalPolicy);
-    }
+    // The RemovalPolicy on the cluster is SNAPSHOT by default, or explicitly configured.
+    // We can safely default to DESTROY for instances that are part of a cluster.
+    //
+    // TODO: This should probably be `DESTROY` always, no, instead of inheriting the
```

Comment:
Definitely should not be `DESTROY` if the Cluster has `RETAIN`...

Reply:
Maybe my understanding of how RDS works is incorrect then.